lib/workers/repository/process/extract-update.ts

```diff
@@ -128,10 +147,6 @@ export async function extract(
         for (const file of files) {
           for (const dep of file.deps) {
             delete dep.updates;
-            if (dep.skipStage && dep.skipStage !== 'extract') {
-              delete dep.skipReason;
-              delete dep.skipStage;
-            }
```

Comment:
When debugging, I found that this was set to `lookup` for cargo extracts. I choose to remove this, instead of attempt to properly implement it